src/explorepy/explore.py

```diff
@@ -146,6 +148,12 @@ def record_data(self, file_name, do_overwrite=False, duration=None, file_type='c
         elif file_type == 'edf':
             self.recorders['marker'] = self.recorders['exg']
 
+        self.recorders['meta'] = create_meta_recorder(filename=meta_out_file,
```

Comment:
We should have similar metadata recorder in `convert_bin` method in this module. In binary file conversion, the device settings might change during the conversion. We can add another record in the metadata if the sampling rate/adc mask has changed in the binary file.